src/main/kotlin/minesweeper/domain/Cell.kt

```diff
@@ -0,0 +1,11 @@
+package minesweeper.domain
+
+data class Cell(val isMine: Boolean) {
+    var isOpen = false
```

Comment:
프로퍼티의 getter 함수를 노출하면서 setter 함수에 대한 접근을 제한하려면 setter 함수를 private으로 지정하면 됩니다.

```suggestion
    var isOpen = false
        private set
```